libraries/cyclestreets-core/src/main/res/values-pt/arrays.xml

```diff
@@ -0,0 +1,125 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <string-array name="units">
+    <item>Kilometres</item>
+    <item>Miles</item>
+  </string-array>
+  <string-array name="unit_values">
+    <item>km</item>
+    <item>miles</item>
+  </string-array>
+  <string-array name="speed">
+    <item>Unhurried 10mph (16km/h)</item>
+    <item>Cruising 12mph (20km/h)</item>
+    <item>Quick 15mph (24km/h)</item>
+  </string-array>
+  <string-array name="speed_values">
+    <item>16</item>
+    <item>20</item>
+    <item>24</item>
+  </string-array>
+  <string-array name="routetype">
+    <item>Quietest</item>
+    <item>Balanced</item>
+    <item>Fastest</item>
+    <item>Shortest</item>
+  </string-array>
+  <string-array name="routetype_values">
+    <item>quietest</item>
+    <item>balanced</item>
+    <item>fastest</item>
+    <item>shortest</item>
+  </string-array>
```

Comment:
@auyer You're exactly right!  You can tell we're new to this translation thing 😄 

I've just pushed a commit which marks the `_values` arrays as `untranslatable`, so you can remove those from your submission.